src/lib/time.ts

```diff
@@ -4,6 +4,6 @@ import { DateTime } from 'luxon';
 export const epochTimeUpcoming = (startDate: DateTime) => {
   return startDate
     .diffNow(['days', 'hours', 'minutes'])
-    .toHuman({ unitDisplay: 'short', notation: 'compact' })
+    .toFormat("d' days, 'h' hr, 'm' mins'")
```

Comment:
Please create `src/lib/time.test.ts` and add some unit tests for the examples shown in #1719, as well as some larger time values that are greater than 1 day.

Reply:
done